customize_example/magento_model.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Author: Guewen Baconnier
+#    Copyright 2013-2015 Camptocamp SA
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+from openerp import models, api
+
+
+class MagentoBackend(models.Model):
+    _inherit = 'magento.backend'
+
+    @api.model
+    def select_versions(self):
```

Comment:
I prefer to keep it for backward compatibility. Also as a general manner I don't use `selection_add` because it does not allow to order the versions.